subsystems/art/TowerVFX.py

```diff
@@ -0,0 +1,82 @@
+import pygame,math
+from pygame.locals import *
+pygame.init()
+class Bullet:
+    def __init__(self,imageName):
+        self.image = pygame.transform.scale(pygame.image.load("%s.jpg" %imageName), (40,40))
+        self.altImage = pygame.transform.scale(pygame.image.load("%s.jpg" %imageName), (40,40))
```

Comment:
again, filepaths get messed up when referenced statically inside classes. This will be handled at a global level. consider YAML